### PR TITLE
Add Slime to integrations

### PR DESCRIPTION
**Pull request by YonkaFang**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/ok-to-test

File: data/companies.yml [hunk 1]
```diff
@@ -228,6 +228,13 @@ integrations:
     description: "Application layer protection for Istio service mesh"
     details:
       - "Provides complete layer 7 protection for microservices architecture applications."
+  - name: "Slime"
+    logo: "/logos/slime.png"
+    url: "https://github.com/slime-io"
+    description: "Slime extends, supports, manages Istio and helps to use Istio right."
+    details:
+      - "Slime has built a manager-modules architecture that makes it easy to organize and manage various extensions which are focused on enhancing the capabilities and improving the ease of use of Istio."
+      - "Slime is like Slime, it can be any shape you want and you can do anything in it that is good for Istio's ecology."
```
Comment (craigbox) [inline data/companies.yml:237]: How about something more like:

```suggestion
    description: "Slime is an intelligent service mesh manager for Istio.
    details:
      - "Slime has a modular architecture that makes it easy to organize and manage various extensions which are focused on enhancing the capabilities and improving the ease of use of Istio."
      - "Using Slime, you can define dynamic service management strategies, which are automatically and conveniently converted to Istio/Envoy configuration."
      - "Just like its namesake, Slime can be any shape you want and it is good for Istio's ecology!"
```

**Review by craigbox — APPROVED** [review]

File: data/companies.yml [hunk 2]
```diff
@@ -228,6 +228,14 @@ integrations:
     description: "Application layer protection for Istio service mesh"
     details:
       - "Provides complete layer 7 protection for microservices architecture applications."
+  - name: "Slime"
+    logo: "/logos/slime.png"
+    url: "https://github.com/slime-io"
+    description: "Slime is an intelligent service mesh manager for Istio.
```
Comment (craigbox) [inline data/companies.yml:234]:
```suggestion
    description: "Slime is an intelligent service mesh manager for Istio."
```

sorry

**Review by craigbox — CHANGES_REQUESTED** [review]

**Review by craigbox — APPROVED** [review]

**Pull request merged**